client/src/components/Ticketing/mainpage/SubscriptionPurchasing/SeasonSubscriptions.tsx

```diff
@@ -0,0 +1,86 @@
+import React, {useState} from 'react';
+import {getSeasonImageDefault} from '../../../../utils/imageURLValidation';
+import {ImageCarousal} from './ImageCarousal';
+import {CarousalEvent} from './CarousalEvent';
+import {SubscriptionOption} from './SubscriptionOption';
+import {SeasonWithSubscriptions} from './SubscriptionPurchaseUtils';
+
+interface SeasonSubscriptionsProps {
+  season: SeasonWithSubscriptions;
+  setPopUpProps: any;
+}
+
+export const SeasonSubscriptions = (props: SeasonSubscriptionsProps) => {
+  const {season, setPopUpProps} = props;
+  const [anchors, setAnchors] = useState(
+    Array(season.events.length).fill(null),
+  );
+
+  return (
+    <main
+      className='bg-fixed bg-cover'
+      style={{
+        backgroundImage: `url(${getSeasonImageDefault(
+          season.imageurl,
+        )}),url(${getSeasonImageDefault()})`,
+      }}
+    >
+      <div
+        className='flex flex-col justify-center items-center min-h-[calc(100vh-233px)]
+                   md:min-h-[calc(100vh-142px)] backdrop-blur-sm
+                   bg-zinc-900/20 px-[1rem] py-[5rem] tab:px-[2rem]'
+      >
+        <section className='grid grid-cols-12 bg-zinc-700/60 min-[1700px]:w-[80%] min-[2000px]:w-[70%] p-2 tab:p-5 rounded-xl md:gap-4 mt-10'>
+          <header className='col-span-12 text-center text-zinc-200 p-2 pb-0'>
+            <h1 className='flex flex-col'>
+              <span className='text-3xl md:text-4xl'>
```

Comment:
One smaller thing, could the span with the `Subscriptions` text be `font-medium`

Reply:
Fixed